### PR TITLE
feat: Merge `Version Control Systems` into same section

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Improve repo

### Why is this valuable (or not)?

- Handles #5526 for `-en` listings

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+
+* [A git Primer](https://danielmiessler.com/study/git/) - Daniel Miessler
+* [A Visual Git Reference](http://marklodato.github.io/visual-git-guide/index-en.html) - Mark Lodato
+* [Conversational Git](http://blog.anvard.org/conversational-git/)
+* [git - the simple guide](http://rogerdudler.github.io/git-guide/)
+* [Git cookbook](https://git.seveas.net/about.html) - Dennis Kaarsemaker (HTML)
+* [Git for Computer Scientists](http://eagain.net/articles/git-for-computer-scientists/)
+* [Git From The Bottom Up](https://jwiegley.github.io/git-from-the-bottom-up/) - J. Wiegley
+* [Git Immersion](http://gitimmersion.com)
+* [Git In The Trenches](http://cbx33.github.io/gitt/index.html)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:820]:
```suggestion
* [Git In The Trenches](http://cbx33.github.io/gitt/index.html) - Peter Savage
```

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Git internals](https://github.com/pluralsight/git-internals-pdf/raw/master/drafts/peepcode-git.pdf) - Scott Chacon (PDF)
+* [Git Magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:822]:
```suggestion
* [Git Magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/) - Ben Lynn et al.
```

File: books/free-programming-books-subjects.md [hunk 3]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Git Notes for Professionals](http://goalkicker.com/GitBook) - Compiled from StackOverflow Documentation (PDF)
+* [Git Pocket Guide](https://www.oreilly.com/library/view/git-pocket-guide/9781449327507) - Richard E. Silverman
+* [Git Reference](https://web.archive.org/web/20170602211147/http://gitref.org/)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:825]:
```suggestion
* [Git Reference](https://web.archive.org/web/20170602211147/http://gitref.org/) *(:card_file_box: archived)*
```

File: books/free-programming-books-subjects.md [hunk 4]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Git Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/git) (PDF, Kindle) (email address *requested*, not required)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:826]:
```suggestion
* [Git Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/git) - Ryan Hodson (PDF, Kindle) (email address *requested*, not required)
```

File: books/free-programming-books-subjects.md [hunk 5]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Git Workflows](https://web.archive.org/web/20210910133251/http://documentup.com/skwp/git-workflows-book) - Yan Pritzker *(:card_file_box: archived)*
+* [Happy Git and GitHub for the useR](https://happygitwithr.com) - Jenny Bryan
+* [Hg Init: a Mercurial Tutorial](https://hginit.github.io) - Joel Spolsky
+* [How to Collaborate on GitHub: A mini book about collaborating on GitHub](https://github.com/eonist/How-to-collaborate-on-github) - André J
+* [Introduction to Git and Github](https://launchschool.com/books/git) - Launch School
+* [Introduction to Git and Github - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
+* [Introduction to Git and GitHub eBook](https://github.com/bobbyiliev/introduction-to-git-and-github-ebook) - Bobby Iliev (Markdown, PDF)
+* [Learn Git - Learn Version Control with Git](http://www.git-tower.com/learn/git/ebook/command-line/introduction) - Tobias Günther
+* [Mercurial: The Definitive Guide](http://hgbook.red-bean.com)
+* [Mercurial: The Definitive Guide 2nd edition](https://book.mercurial-scm.org)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:838]:
```suggestion
* [Mercurial: The Definitive Guide](http://hgbook.red-bean.com) - Bryan O'Sullivan
* [Mercurial: The Definitive Guide 2nd edition](https://book.mercurial-scm.org) - Bryan O'Sullivan
```

File: books/free-programming-books-subjects.md [hunk 6]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Pro Git](http://git-scm.com/book/en/v2) - Scott Chacon
+* [Pro Git Reedited](https://leanpub.com/progitreedited/read) - Jon Forrest
+* [Ry's Git Tutorial](https://web.archive.org/web/20161121145226/http://rypress.com:80/tutorials/git/index) - Ryan Hodson
+* [Subversion Version Control](http://ptgmedia.pearsoncmg.com/images/0131855182/downloads/Nagel_book.pdf) (PDF)
+* [Think Like (a) Git: A Guide for the Perplexed](http://think-like-a-git.net)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:843]:
```suggestion
* [Ry's Git Tutorial](https://web.archive.org/web/20161121145226/http://rypress.com:80/tutorials/git/index) - Ryan Hodson *(:card_file_box: archived)*
* [Subversion Version Control](http://ptgmedia.pearsoncmg.com/images/0131855182/downloads/Nagel_book.pdf) - William Nagel (PDF)
* [Think Like (a) Git: A Guide for the Perplexed](http://think-like-a-git.net) - Sam Livingston-Gray
```

File: books/free-programming-books-subjects.md [hunk 7]
```diff
@@ -806,6 +807,43 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Version Control with Subversion](http://svnbook.red-bean.com)
```
Comment (LuigiImVector) [inline books/free-programming-books-subjects.md:844]:
```suggestion
* [Version Control with Subversion](https://svnbook.red-bean.com/index.en.html) - Ben Collins-Sussman, Brian W. Fitzpatrick, C. Michael Pilato
```

This resource is in italian (#6934) but there is a correct link in english

Comment (davorpa) [reply books/free-programming-books-subjects.md:844]: Language is took from browser automatically. I see it in spanish. In anyway, it's safe take the right `index.html` as like as you catch it

**Comment by davorpa** [comment]
I'm not in pro of mix PR topics but in anyway was simple to apply suggestions. Thanks @LuigiImVector

**Review by LuigiImVector — APPROVED** [review]

**Pull request merged**